route/queries.js

```diff
@@ -2,13 +2,15 @@ import qql from 'graphql-tag';
 
 /*
  * In this example we request a route from Amsterdam, Netherlands to Berlin, Germany
+ * Your origin and destination are required fields. You also need to select an EV.
+ * Only the EV ID here is mandatory, all other fields are optional and when not specified will use the default values.
  * The changing conditions are:
  *   - full battery at Amsterdam, Netherlands
  *   - no desired range at Berlin, Germany
  *   - EV can charge at CHadMO changers
```

Comment:
no desired range is incorrect (maybe we should just remove the battery on the request and use the defaults)

Reply:
I removed the sentence no desired range